docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -6,6 +6,16 @@ The Oracle connector allows querying and creating tables in an external Oracle
 database. Connectors let Trino join data provided by different databases,
 like Oracle and Hive, or different Oracle database instances.
 
+Requirements
+------------
+
+Requirements for using the connector in a catalog to connect to a Oracle data
```

Comment:
GDDSD suggests the use of [active voice](https://developers.google.com/style/voice?hl=en):

To connect to an Oracle data source, the following requirements must be met:

Reply:
I'd avoid "must be", what about:

"To connect to an Oracle data source, you need:"

or

"Before you connect to an Oracle data source, you need:"